remote/server.go

```diff
@@ -23,7 +22,7 @@ var rootContext = actor.EmptyRootContext
 
 // Start the remote server
 func Start(address string, options ...RemotingOption) {
-	grpclog.SetLogger(slog.New(ioutil.Discard, "", 0))
+	grpclog.SetLoggerV2(grpclog.NewLoggerV2(ioutil.Discard, ioutil.Discard, ioutil.Discard))
```

Comment:
5df75ed replaces the usage of the deprecated method.
Since the previous logger used `ioutil.Discard`, replacing code also uses `ioutil.Discard` for all levels.
If this is preferable to set separate `io.Writer` for each log level, that can be done in the following pull request.